### ISSUE TITLE
ReferenceError: SimpleItemSheet is not defined

### ISSUE THREAD

**Issue by pandrien** [ElusiveZenith/fvtt-laf#2]
init.js imports SimpleActorSheet but never imports SimpleItemSheet

I think init.js needs a line like this at the top:
`import { SimpleItemSheet} from "./item-sheet.js";`

**Comment by aniki21** [comment]
Adding this causes an error as the `ATTRIBUTE_TYPES` constant and the `constants.js` file don't exist. Creating the file and exporting an empty `const ATTRIBUTE_TYPES` array also throws an error, on line 26 of the SimpleItemSheet.

I don't know enough about the intended behaviour here to fully diagnose and fix the issue, though.

**Comment by ElusiveZenith** [comment]
Thanks for pointing it out. There was a bit of a scorched earth approach that I took in migrating to the latest version of foundry when I took this over, this apparently was one piece that I overlooked.

I poked around and got it to load, but there are issues with updating the fields which I know was one thing that was broken in v10 and had to migrate for the Actor sheet. I will have to look back when I have time to figure out what I did before.

Also, sorry for the very late response. I didn't have the notifications properly set up for this repo and didn't see it. I just happened to check in on this repo today.
